airbyte-integrations/connectors/source-mixpanel/pyproject.toml

```diff
@@ -3,7 +3,7 @@ requires = [ "poetry-core>=1.0.0",]
 build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
-version = "2.3.0"
+version = "2.3.1"
```

Comment:
this needs to be a breaking change because we're resetting the connection state

Reply:
I have updated it to 3.0.0 now